src/components/EventLog.tsx

```diff
@@ -23,11 +24,12 @@ export default class EventLog extends React.Component<Props> {
     }
 
     render() {
-        const { entries } = this.props;
+        const { entries, hideLog } = this.props;
         const latest = entries.slice().reverse();
-        return (
-            <LabeledPanel title="Event Log" id="event-log">
-                <div ref={el => (this.dummy = el)}/>
+
+        let content = null;
+        if (!hideLog) {
```

Comment:
Would be nice to have something that looks nicer (e.g. an overlay saying "You clear your mind from distractions")

Reply:
opened an issue about that already: #36